Replacement of font size variables

GusBaamonde opened the pull request:
I removed the old font size variables, replacing them with new variables in levels, impacting all the scss files. I've talked to Santiago Paolucci and he agrees.

![variables-font-size](https://user-images.githubusercontent.com/46735526/55994989-afba2e00-5c89-11e9-94d2-da06af7ecea4.gif)

andresmoschini commented on assets/scss/core/_variables.scss:
```diff
@@ -25,26 +26,33 @@ $dp-font-color-dark: $dp-font-color !default;
+/// Font Size settings
+///
+
+$dp-font-size--lv0: 0;
```
Does this variable make sense?

GusBaamonde replied:
En algunos casos necesitamos ocultar el contenido, actualmente pasa con el logo en signup, el navegador lo lee pero no tiene tamaño el contenido y se coloca la imagen por background

andresmoschini replied:
Thanks for the explanation, you are the expert.

andresmoschini commented:
It is fine for me, maybe @mvillaseco would like to review it.

cbernat commented on assets/scss/core/_variables.scss:
```diff
@@ -25,26 +26,33 @@ $dp-font-color-dark: $dp-font-color !default;
+$dp-font-size--lv3: 12px;
+$dp-font-size--lv4: 13px;
+$dp-font-size--lv5: 14px;
+$dp-font-size--lv6: 15px;
```
Do we use all this fontsizes for real? Seems a bit too much

mvillaseco replied:
Omg, 19 font sizes? are we using them as other things also?(maybe a rename should be done)

andresmoschini replied:
I agree as we have standardized spaces for common usages, we should have standardized font sizes for common usages.

damianmuti commented:
Despite the fact that we all agreed to have levels for sizes, it seems that this approach lacks of any sort of description in the naming.
Just as an exercise for anyone that looks at this comment, try to think about what's the font size for buttons with this new naming convention (without scrolling up). No clue? Me neither. 😝

IgnacioRodrigues commented:
This nomenclature of variables seems not to be the correct one, like knowing what level to use in each case? Support to what they say in which the nomenclatures tell you to use them, with this change you lose the sense of using classes.

GusBaamonde commented:
![fonts](https://user-images.githubusercontent.com/46735526/56148358-7f6fd980-5f80-11e9-8e98-e5b9318d2e5f.gif)


**To unify criteria of variables, I talked with Santiago Paolucci, @mvillaseco  and @damianmuti and I think we reached a midpoint.**

- I made a survey of the size fonts currently used.
- I reduced the quantity of size fonts used.
- We will use the guide already posed in the mockup of the library for future designs (we talk with Santiago Paolucci and compare with the fonts of the mockup of the library).
- We will use more global levels (We speak with @mvillaseco ).
- We will not have specific names at the moment. (We talked to @damianmuti , and it's a good idea to make 'variables of variables' in order to make themes, improve for the future)
- I changed and cleaned all the font size variables we are using.

andresmoschini commented:
Looks fine!

andresmoschini commented:
:tada: This PR is included in version 1.0.0 :tada:

The release is available on [GitHub release](https://github.com/FromDoppler/Doppler-UI-System/releases/tag/v1.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
